src/pages/Login.jsx

```diff
@@ -0,0 +1,67 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import React, { useState } from "react";
+import classes from "../css/Login.module.css";
+import { auth } from "./../firebase/firebase";
+import { Link, Navigate } from "react-router-dom";
+import { paths } from "../app/Route";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await signInWithEmailAndPassword(auth, email, password);
```

Comment:
Pre slanja request-a na BE potrebno je odraditi validaciju ( ne uvek ali u ovom slucaju da ). Uvek je dobro da proveris makar da li su polja prazna. Ako jesu, prikazi gresku korisniku